nautobot/core/tests/test_graphql.py

```diff
@@ -1006,21 +1006,23 @@ def test_query_circuit_termination_connected_endpoint(self):
     @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
     def test_query_config_context_and_custom_field_data(self):
 
-        query = """
-        query {
-            devices {
-                name
-                config_context
-                _custom_field_data
-            }
-            device (id: "%s") {
-                name
-                config_context
-                _custom_field_data
-            }
-        }
-        """ % (
-            self.device1.id,
+        query = (
+            ""  # pylint: disable=C0209
+            """
```

Comment:
What's going on here?

Reply:
Sorry for some reason I thought the pylint comment will only be useful if it is in the middle of the multiline string and that was my weird way of inserting the comment. Turns out as long as it is in the parentheses, it will be fine.